requirements.txt

```diff
@@ -1,4 +1,6 @@
-# This can not be installed from pip, on Fedora you need to install rpm-python{,3}
+# This can not be installed from pip, on Fedora you need to install
+# python{2,3}-rpm RPM including rpm-python pip package.
```

Comment:
According to your [comment](https://github.com/rebase-helper/rebase-helper/pull/286#issuecomment-312655923), I think it would be best to say something like: `This can not be installed from pip, on Fedora you need to install python{2,3}-rpm package containing the module, see:` followed by the link.
And after the module is renamed later we will change `#rpm-python` to just `#rpm`.